lib/move-bindgen-derive/src/lib.rs

```diff
@@ -36,23 +35,23 @@ fn derive_type_tagged(
         });
     where_clause.predicates.extend(bounds.clone());
 
-    let [module] = attrs
-        .into_iter()
-        .filter(|attr| attr.path().is_ident("move_output_type"))
-        .map(|attr| {
-            let mnv =
-                syn::parse2::<MetaNameValue>(attr.meta.require_list().unwrap().tokens.clone())
-                    .unwrap();
-
-            assert!(mnv.path.is_ident("module"));
-            match &mnv.value {
-                Expr::Path(expr) => expr.path.require_ident().unwrap().clone(),
-                _ => panic!("???"),
-            }
-        })
-        .collect::<Vec<_>>()
-        .try_into()
-        .unwrap();
+    // let [module] = attrs
+    //     .into_iter()
+    //     .filter(|attr| attr.path().is_ident("move_output_type"))
+    //     .map(|attr| {
+    //         let mnv =
+    //             syn::parse2::<MetaNameValue>(attr.meta.require_list().unwrap().tokens.clone())
+    //                 .unwrap();
+
+    //         assert!(mnv.path.is_ident("module"));
+    //         match &mnv.value {
+    //             Expr::Path(expr) => expr.path.require_ident().unwrap().clone(),
+    //             _ => panic!("???"),
+    //         }
+    //     })
+    //     .collect::<Vec<_>>()
+    //     .try_into()
+    //     .unwrap();
```

Comment:
intented to be keeped?

Reply:
yeah I need to figure out what to do with this